sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -310,12 +307,11 @@ def GetPartitionResolver(self, database_link: str) -> Optional[RangePartitionRes
         """
         if not database_link:
             raise ValueError("database_link is None or empty.")
-
         return self.partition_resolvers.get(base.TrimBeginningAndEndingSlashes(database_link))
 
     def CreateDatabase(
         self,
-        database: Mapping[str, Any],
+        database: Dict[str, Any],
```

Comment:
is there a particular reason to switch over to a `Dict` ?

Reply:
We call `json.dumps` on the request payload. The json library does not support the `Mapping` protocol.